styles.css

```diff
@@ -568,3 +536,619 @@ footer {
+  }
+
+  /** About me SECTION **/
+  .about-me {
+    display: flex;
+    flex-direction: column;
+    margin-top: 80px;
+    background-color: #1c1a19;
+    color: #f4f5f7;
+    background-image: url("img/about-me.png");
+    background-repeat: no-repeat;
+    background-size: 450px;
+    background-position: right 80px;
+  }
+
+  .about-me > .aboutme-flex {
+    display: flex;
+    flex-direction: column;
+    width: 100vh;
+    padding-right: 230px;
+  }
+
+  .about-me > .aboutme-flex > h1 {
+    float: left;
+    color: #ff6b00;
+    padding-top: 30px;
+    padding-bottom: 20px;
+    font-family: 'Crete Round', sans-serif;
+    font-weight: 400;
+    font-size: 72px;
+  }
+
+  .about-me > .aboutme-flex > h3 {
+    float: left;
+    color: white;
+    padding-bottom: 5px;
+    width: 327px;
+    text-align: center;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  .aboutme-flex > .introduction {
+    float: left;
+    color: white;
+    padding-bottom: 30px;
+    width: 327px;
+    text-align: left;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 20px;
+  }
+
+  .about-me > .aboutme-flex > button {
+    float: left;
+    color: #f4f5f7;
+    background-color: #ff6b00;
+    border: none;
+    align-items: left;
+    text-align: center;
+    padding: 12px;
+    font-family: 'Inter', sans-serif;
+    margin-bottom: 50px;
+    max-width: fit-content;
+  }
+
+  .about-me > button:hover {
+    background: #fff5e1;
+    border: 1px solid #f55800;
+    color: #f55800;
+  }
+
+  .line {
+    margin-top: 70px;
+    border: 1px solid #fff;
+    width: 100vh;
+    margin-bottom: 80px;
+  }
+
+  .header-grid {
+    display: grid;
+    grid-template-columns: 30% 30% 30%;
+    gap: 120px;
+    padding-right: 150px;
+  }
+
+  .header-grid > .header-flex {
+    display: flex;
+    flex-direction: row;
+    padding-bottom: 20px;
+    font-family: 'InaiMathi', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 30px;
+    line-height: 44px;
+    align-items: left;
+    gap: 5px;
+    color: #fff;
+  }
+
+  .about-me-list {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 43px;
+    display: flex;
+    flex-direction: column;
+    align-items: baseline;
+    color: #fff;
+    list-style: none;
+    margin-bottom: 40px;
+  }
+
+  /** Form SECTION **/
+  #contact-form {
+    padding: 120px 270px;
+    display: flex;
+    flex-direction: row;
+    border-bottom: 1px solid rgb(219, 216, 215);
+  }
+
+  #contact-from * {
+    box-sizing: border;
+  }
+
+  #contact-form h2 {
+    font-family: 'Inter', sans-serif;
+    font-size: 32px;
+    color: rgb(23, 43, 77);
+    line-height: 44px;
+    text-align: left;
+    font-weight: 700;
+    margin-bottom: 58px;
+    width: 500px;
+  }
```

Comment:
The width, height, font-size, etch of the contact form text is different form that of the figma design. These disparities has created a dilfference between your design and that of the figma. Please your design will be more beautiful and proffessional if you could work on that.

Your design             |  Figma design
:-------------------------:|:-------------------------:
![Screenshot from 2022-09-29 20-52-32](https://user-images.githubusercontent.com/65233860/193145277-ea5d8e60-010a-46ad-9ef1-59250e4526cf.png)  |  ![Screenshot from 2022-09-29 20-52-25](https://user-images.githubusercontent.com/65233860/193145291-5dfcbb31-2f71-4b37-8097-f4dceec787b6.png)